### PR TITLE
[semver:patch] Set default minor version of the agent to latest

### PR DESCRIPTION
**Pull request by TovRudyy**

### CODE REVIEW

File: src/commands/setup.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ parameters:
   agent_minor_version:
     type: string
     description: The minor version of the agent to use
-    default: "54.1"
+    default: "latest"
```
Comment (spencergilbert) [inline src/commands/setup.yml:14]:
```suggestion
    default: ""
```

File: README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ jobs:
       - checkout
       - datadog-agent/setup:
           agent_major_version: "7"
+          agent_minor_version: "latest"
```
Comment (spencergilbert) [inline README.md:23]:
```suggestion
```

**Review by spencergilbert — CHANGES_REQUESTED** [review]
Sorry, miscommunicated I think - as-is this would try and install a `7.latest` version. `agent_minor_version: ""` skips the minor version and picks the latest `7.x`.

**Review by spencergilbert — APPROVED** [review]

**Review by Pythyu — APPROVED** [review]
👍

**Review by rodrimendoza — APPROVED** [review]

**Comment by rodrimendoza** [comment]
> Sorry, miscommunicated I think - as-is this would try and install a `7.latest` version. `agent_minor_version: ""` skips the minor version and picks the latest `7.x`.

An inline comment with this would be nice.

**Pull request merged**